website/docs/addon_3dequalizer_artist.md

```diff
@@ -21,3 +21,53 @@ import versions from '@site/docs/assets/json/Ayon_addons_version.json'
 - [Manage (Inventory)](artist_tools_inventory)
 - [Publish](artist_tools_publisher)
 - [Library Loader](artist_tools_library_loader)
+
+
+## Overview
+
+3DEqualizer (3DE) Addon allows you to manage your 3DE projects, load plates as cameras and publishing of scripts for Nuke and Maya with Lens Distortion node for Nuke.
+
+### Loading plates
+
+You can load image data into 3DE by using [Loader]((artist_tools_loader)) tool. Go to ```AYON``` menu in 3DE, select ```Load``` and pick your image sequence you want yo use. This will create new sequence camera, set the path for the image sequence and set correct frame range. From that moment loaded sequence is version tracked, you can follow used versions in [Manage tool](artist_tools_inventory).
+
+:::note
+AYON is using **Project Notes** to store some of its infomations for version tracking. If you want to use them, please, do not modify content of ```AYON_CONTEXT:: ... ::AYON_CONTEXT``` block as it will probably break version management of loaded image sequences.
+:::
+
+### Publishing Matchmove scripts
+ 
+Once you track your camera in 3DE, you can export it and publish it as version for Maya and/or Nuke. Just go yo ```AYON``` menu in 3DE, click on [Publish](artist_tools_publisher). You can create ```Match Move``` publishing instance. There, you can set bunch of settings:
+
+What cameras to publish (**Camera(s) to publish**). You can select eirther *All Cameras*, *Current Camera* (active), *Reference Cameras* (for all reference cameras) and *Sequence Cameras*.
```

Comment:
```suggestion
What cameras to publish (**Camera(s) to publish**). You can select either *All Cameras*, *Current Camera* (active), *Reference Cameras* (for all reference cameras) and *Sequence Cameras*.
```

Reply:
Thank you. it should be resolved by https://github.com/ynput/ayon-documentation/pull/245/commits/684846a53fd4d2a6400a20db9309d3f8f6f97b6c